modules/install/pages/upgrade.adoc

```diff
@@ -23,6 +23,31 @@ See xref:install:upgrade-procedure-selection.adoc#swap-rebalance[Swap Rebalance]
 
 Before upgrading, consider the following version compatibility concerns.
 
+[#8-0-storage-backend]
+=== New Default Storage Backend in Couchbase Server Version 8.0
+
+After you have fully upgraded a cluster to Couchbase Server 8.0.x and later, the default storage backend for buckets is Magma with 128 vBuckets.
+Previous versions of Couchbase Server used Couchstore with 1024 vBuckets as the default storage backend.
+
+This new default results in two behavior changes from previous versions:
+
+* If you create a bucket and do not specify the storage backend, your bucket will use the Magma storage backend instead of the Couchstore backend.
+* If you specify Magma as the storage backend but do not set the new `numVBuckets` parameter, the bucket will have 128 vBuckets instead of the prior default of 1024 vBuckets.
+Magma buckets with 128 vBuckets is a new feature in Couchbase Server 8.0 and later.
+
+These behavior changes could cause issues if you rely on the prior behavior, especially if you use deployment scripts.
+If you have deployment scripts that create buckets, review them to determine if you need to make changes.
+
+For example, suppose your deployment script does not specify the storage backend when it creates a bucket that you intend to use with the xref:views/views-mapreduce-intro.adoc[] feature.
```

Comment:
Since we are giving a link to the MapReduce Views page here, could we have the same note that we have in Views intro page saying that Views are deprecated and will be removed in a future version in the MapReduce Views page?

Link to Views intro page --[ ](https://docs.couchbase.com/server/current/learn/views/views-intro.html)